permission-portal-frontend/client/src/store/index.js

```diff
@@ -50,15 +49,15 @@ const Organization = types
     verificationNotSharedText: types.maybeNull(types.string),
     diagnosisText: types.string,
     exposureText: types.string,
-    members: types.maybeNull(types.array(User))
+    members: types.maybeNull(types.array(User)),
   })
-  .actions(self => {
+  .actions((self) => {
     const store = getParent(self)
     const setOrganizationalBranding = flow(function* (diagnosisText, exposureText) {
       try {
         yield firebase.updateOrgTexts(store.user.organizationID, {
```

Comment:
Just noticed the exceptions aren't flowing through due to the try catch statements. These setters / getters return promise objects, the pages use the successs / failure of the promise to determine which toast to show. If the exception is caught, the promise will always resolve as a success.

Reply:
You can test this out manually by throwing in one of these operations and you should see the failure toast pop up

Reply:
Ok, I've changed it so that errors are being passed through. Afaict `setOrganizationalBranding` is broken. Can you check the changes I made to it, and `firebase.updateOrgTexts`?

Reply:
When I hit save on the Organizational Branding page, I get
```
Branding data failed to save: TypeError: Cannot read property 'setOrganizationalBranding' of null
```.
That function is a `store.organization` function, suggesting that organization is not being properly set in the `store`.

Reply:
#93 fixes that